tfhe/src/core_crypto/gpu/mod.rs

```diff
@@ -43,6 +42,22 @@ impl CudaStreams {
             gpu_indexes,
         }
     }
+    /// Create a new `CudaStreams` structure with the GPUs with id provided in a list
+    pub fn new_multi_gpu_with_indexes(indexes: &[GpuIndex]) -> Self {
+        let _gpu_count = setup_multi_gpu();
```

Comment:
You shouldn't need this, no?

Reply:
I remember you saying setup_multi_gpus() should always be called before a multi gpu execution. I don't remember what that method though. Can I remove that line?